codegen.mts

```diff
@@ -12,7 +12,10 @@ const config: CodegenConfig = {
         typeDefsFilePath: false,
         mergeSchema: {
           path: '../../../../schema.graphql',
-          config: { includeDirectives: true },
+          config: {
+            includeDirectives: true,
+            append: '\n\n directive @oneOf on INPUT_OBJECT',
```

Comment:
Retaining the `@oneOf` directive (which is not yet part of the graphql specification), which is not yet supported by GraphQL federation, but shipped with `graphql-js` by default is a fever dream. The easiest solution to retain it for the output schema that we publish to the schema registry is to manually append it, which I achieved by patching codegen...

Once we support `@oneOf` within Federation natively, we can remove this part.

Reply:
anything @eddeee888 can help with here?

Reply:
no really its not worth putting time into